app/models/asset.rb

```diff
@@ -323,6 +323,10 @@ def proxy_start_time
  def ams1_legacy_metadata
     ams1_legacy_metadata ||= find_annotation_attribute("ams1_legacy_metadata")
   end
+   
+ def note
+    note ||= find_annotation_attribute("note")
+  end
```

Comment:
Unindent `end` by one space to match it's opening `def` (same for `ams1_legacy_metadata` above, and any other place you might see where they are unaligned).